### PR TITLE
UI and API routes, and constants

### PR DESCRIPTION
**Pull request by pooja-koul-incontact**

### CODE REVIEW

File: src/app/PSAUIRoutes.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+export class PSAUIRoutes {
```
Comment (akash-nema-incontact) [inline src/app/PSAUIRoutes.ts]: You can add below constants to PSARoutes file and add needed permissions as given for other routes. If already given, please verify them.

File: src/app/bu/bu-rule-mapping/bu-display-rule-mapping/bu-rule-mapping.component.ts [hunk 2]
```diff
@@ -150,8 +151,8 @@ export class BuRuleMappingComponent implements OnInit {
   }
   onClickDismissModal() {
     this.showErrorModalDialog = false;
-    this.router.navigateByUrl('tenants/tenant-details/saved-mapping', {skipLocationChange: true}).then(() =>
-      this.router.navigate(['tenants/tenant-details/rule-mapping']));
+    this.router.navigateByUrl(PSAUIRoutes.ROUTING_MODULE_SAVED_USER_MAPPING, {skipLocationChange: true}).then(() =>
```
Comment (akash-nema-incontact) [inline src/app/bu/bu-rule-mapping/bu-display-rule-mapping/bu-rule-mapping.component.ts]: Refactor all Routes constants from PSARoutes

File: src/app/bu/bu-rule-mapping/service/bu-rule-mapping.service.ts [hunk 3]
```diff
@@ -53,24 +36,42 @@ export class BuRuleMappingService {
...
-    return this.http.put('/psa/v1/rule/assign', assignRuleObject);
+    return this.http.put(PSAAPIRoutes.RULE_MAPPING_ASSIGN_RULE_TO_USER_URI, assignRuleObject);
   }
 
   updateRule(updateRuleObject, icBuUuid, ruleUuid) {
-    return this.http.patch('/psa/v1/rule/mapping/' + icBuUuid + '/' + ruleUuid, updateRuleObject);
+    return this.http.patch(PSAAPIRoutes.RULE_MAPPING_UPDATE_RULE_URI + icBuUuid + '/' + ruleUuid, updateRuleObject);
+  }
+
+  getUsersAssignedToRule(icBUUUID, ruleUuid) {
```
Comment (akash-nema-incontact) [inline src/app/bu/bu-rule-mapping/service/bu-rule-mapping.service.ts]: Change name to icBuUUID

File: src/app/PSAUIRoutes.ts (hunk 1 above)
Comment (pooja-koul-incontact) [reply src/app/PSAUIRoutes.ts]: done

File: src/app/bu/bu-rule-mapping/bu-display-rule-mapping/bu-rule-mapping.component.ts (hunk 2 above)
Comment (pooja-koul-incontact) [reply src/app/bu/bu-rule-mapping/bu-display-rule-mapping/bu-rule-mapping.component.ts]: done

File: src/app/bu/bu-rule-mapping/service/bu-rule-mapping.service.ts (hunk 3 above)
Comment (pooja-koul-incontact) [reply src/app/bu/bu-rule-mapping/service/bu-rule-mapping.service.ts]: changed name

**Pull request merged**